kit/src/main/java/com/oracle/javafx/scenebuilder/kit/util/CssInternal.java

```diff
@@ -59,18 +59,16 @@
 import com.oracle.javafx.scenebuilder.kit.editor.EditorPlatform.Theme;
 import com.oracle.javafx.scenebuilder.kit.fxom.FXOMInstance;
 import com.oracle.javafx.scenebuilder.kit.metadata.property.ValuePropertyMetadata;
-import javafx.css.CompoundSelector;
 import javafx.css.Rule;
 import javafx.css.Selector;
-import javafx.css.SimpleSelector;
 import javafx.css.Style;
 import javafx.css.Stylesheet;
 import javafx.css.CssParser;
 
 /**
  *
- * Utility classes using css internal classes (from com.sun package). Note that
- * the CSS Analyzer is also using extensively com.sun classes.
+ * Utility classes to extract CSS information.
+ * Note: Requires JavaFX 23+
  *
  */
 public class CssInternal {
```

Comment:
Hi @jperedadnr , for me how SceneBuilder internally handles CSS is at this moment not very transparent. Especially with having the CSS inspector not working (since version 9), it would be good to have basic tests attached which illustrate how Scene Builder handles CSS and how things are supposed to work.

Is it possible that you could add a minimalistic test here with a bried description how things should work?
We could use this as a starting point for an architectural description / documentation.

Reply:
I was looking over it and indeed, its all private API. Thats not the ideal place to start integration of tests. Technically this seems to work. So this is actually okay for me.